.vscode/settings.json

```diff
@@ -34,5 +34,15 @@
         "/</\\w+>/g",
         "/SR\\d{4}/g",
         "/[A-Z]{3}[12]/g"
-    ]
+    ],
+    "[html]": {
+        "editor.formatOnSave": false
+    },
+    "[markdown]": {
+        "editor.formatOnSave": false,
+        "files.trimTrailingWhitespace": false
+    },
+    "[scss]": {
+        "editor.formatOnSave": false
+    }
```

Comment:
Do we really want to change these in this PR? Suggest hoisting these changes to a separate PR. (I'm also unsure we want to set `formatOnSave` here explicitly? `trimTrailingWhitespace` should probably be in the `.editorconfig` rather than here, though I kinda think it should be enabled most of the time)

Reply:
Sorry committed these local changes by mistake, have force pushed away